src/pages/PageProfile.vue

```diff
@@ -73,6 +74,13 @@ export default {
     showPreviewAs() {
       return this.variables.username === null && this.editing === null;
     },
+    allowStaffOnlyFields() {
+      console.log(this.viewAs.filter);
```

Comment:
Think you missed this, otherwise, looks good :+1:

Reply:
:+1: